test/Sentry.Extensions.Logging.Tests/SentryLoggerTests.cs

```diff
@@ -124,6 +124,22 @@ public void LogCritical_MatchingFilter_DoesNotCapturesEvent()
                 .CaptureEvent(Arg.Any<SentryEvent>());
         }
 
+        [Fact]
+        public void LogCritical_MatchingFilter_DoesNotAddBreadcrumb()
```

Comment:
`l`: We could add a comment to reference the issue this test reproduces.

Reply:
I'm not sure it adds much value to decorate the unit tests with the issue link or number though.
Have you found this to be useful when browsing code?